modular_skyrat/modules/emote_panel/code/emote_panel.dm

```diff
@@ -0,0 +1,987 @@
+	set name = "~ Gag"
+	set category = "Emotes"
+	usr.emote("gag", intentional = TRUE)
+
+/mob/living/proc/emote_giggle()
+	set name = "~ Giggle"
+	set category = "Emotes"
+	usr.emote("giggle", intentional = TRUE)
+
+/mob/living/proc/emote_glare()
+	set name = "~ Glare"
+	set category = "Emotes"
+	usr.emote("glare", intentional = TRUE)
+
+/mob/living/proc/emote_grin()
+	set name = "~ Grin"
+	set category = "Emotes"
+	usr.emote("grin", intentional = TRUE)
+
+/mob/living/proc/emote_groan()
+	set name = "~ Groan"
+	set category = "Emotes"
+	usr.emote("groan", intentional = TRUE)
+
+/mob/living/proc/emote_grimace()
+	set name = "~ Grimace"
+	set category = "Emotes"
+	usr.emote("grimace", intentional = TRUE)
+
+/mob/living/proc/emote_jump()
+	set name = "~ Jump"
+	set category = "Emotes"
+	usr.emote("jump", intentional = TRUE)
+
+/mob/living/proc/emote_kiss()
+	set name = "| Kiss |"
+	set category = "Emotes"
+	usr.emote("kiss", intentional = TRUE)
+
+/mob/living/proc/emote_laugh()
+	set name = "> Laugh"
+	set category = "Emotes"
+	usr.emote("laugh", intentional = TRUE)
+
+/mob/living/proc/emote_look()
+	set name = "~ Look"
+	set category = "Emotes"
+	usr.emote("look", intentional = TRUE)
+
+/mob/living/proc/emote_nod()
+	set name = "~ Nod"
+	set category = "Emotes"
+	usr.emote("nod", intentional = TRUE)
+
+/mob/living/proc/emote_point()
+	set name = "~ Point"
+	set category = "Emotes"
+	usr.emote("point", intentional = TRUE)
+
+/mob/living/proc/emote_pout()
+	set name = "~ Pout"
+	set category = "Emotes"
+	usr.emote("pout", intentional = TRUE)
+
+/mob/living/proc/emote_scream()
+	set name = "> Scream"
+	set category = "Emotes"
+	usr.emote("scream", intentional = TRUE)
+
+/mob/living/proc/emote_scowl()
+	set name = "~ Scowl"
+	set category = "Emotes"
+	usr.emote("scowl", intentional = TRUE)
+
+/mob/living/proc/emote_shake()
+	set name = "~ Shake"
+	set category = "Emotes"
+	usr.emote("shake", intentional = TRUE)
+
+/mob/living/proc/emote_shiver()
+	set name = "~ Shiver"
+	set category = "Emotes"
+	usr.emote("shiver", intentional = TRUE)
+
+/mob/living/proc/emote_sigh()
+	set name = "> Sigh"
+	set category = "Emotes"
+	usr.emote("sigh", intentional = TRUE)
+
+/mob/living/proc/emote_sit()
+	set name = "~ Sit"
+	set category = "Emotes"
+	usr.emote("sit", intentional = TRUE)
+
+/mob/living/proc/emote_smile()
+	set name = "~ Smile"
+	set category = "Emotes"
+	usr.emote("smile", intentional = TRUE)
+
+/mob/living/proc/emote_sneeze()
+	set name = "> Sneeze"
+	set category = "Emotes"
+	usr.emote("sneeze", intentional = TRUE)
+
+/mob/living/proc/emote_smug()
+	set name = "~ Smug"
+	set category = "Emotes"
+	usr.emote("smug", intentional = TRUE)
+
+/mob/living/proc/emote_sniff()
+	set name = "> Sniff"
+	set category = "Emotes"
+	usr.emote("sniff", intentional = TRUE)
+
+/mob/living/proc/emote_stare()
+	set name = "~ Stare"
+	set category = "Emotes"
+	usr.emote("stare", intentional = TRUE)
+
+/mob/living/proc/emote_strech()
+	set name = "~ Stretch"
```

Comment:
what's the difference between the >, ~, and | | emotes?

Reply:
That's basically only using for ordering. 
> Emote has sound effect
| Emote | is an "action" (flip, kiss)
< Emote > is race specific (basically only used on synth, silicon and brain. Could be used on simple ons in the future)
~ Emote is a regular text-only emote